### PR TITLE
 Add sample of JSON output from my GET API

### PR DESCRIPTION
**Pull request by camper2018**

### CODE REVIEW

File: client/app.jsx [hunk 1]
```diff
@@ -16,6 +15,7 @@ class App extends Component {
       const id = Number(numbersArr.join(""));
       if (id >= 0 && id <= 100) {
         $.get(`/${id}`, (data)=> {
+          console.log('data:', data)
```
Comment (creationator) [inline client/app.jsx]: You can delete this console.

File: client/components/form.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      quantity: null,
+      country: null,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    console.log('event' ,event);
```
Comment (creationator) [inline client/components/form.jsx]: Delete this one as well.

File: client/app.jsx (hunk 1 above)
Comment (camper2018) [reply client/app.jsx]: Did it , thanks.

File: client/components/form.jsx (hunk 2 above)
Comment (camper2018) [reply client/components/form.jsx]: Done.Thanks

**Pull request merged**